ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -311,13 +311,43 @@ extension SignalProducerType {
 	}
 
 	/// Creates a Signal from the producer, then adds exactly one observer to
-	/// the Signal, which will invoke the given callbacks when events are
+	/// the Signal, which will invoke the given callback when `next` events are
 	/// received.
 	///
 	/// Returns a Disposable which can be used to interrupt the work associated
 	/// with the Signal, and prevent any future callbacks from being invoked.
-	public func start(error error: (E -> ())? = nil, completed: (() -> ())? = nil, interrupted: (() -> ())? = nil, next: (T -> ())? = nil) -> Disposable {
-		return start(Event.sink(next: next, error: error, completed: completed, interrupted: interrupted))
+	public func startNext(next: T -> ()) -> Disposable {
+		return start(Event.sink(next: next))
+	}
+
+	/// Creates a Signal from the producer, then adds exactly one observer to
+	/// the Signal, which will invoke the given callback when an `error` event is
+	/// received.
+	///
+	/// Returns a Disposable which can be used to interrupt the work associated
+	/// with the Signal.
+	public func startError(error: E -> ()) -> Disposable {
```

Comment:
The whole point of calling them `startNext`, etc. was to avoid the overload ambiguity while still allowing trailing closure syntax. Forcing you to name the params is going to break that and force an extra set of parens. I'm open to better names, but I think the `observe*` ones are good and am ok with the `start*` variants even if there not perfect it if means trailing closures and consistency.